core/src/main/java/io/onetable/delta/DeltaValueConverter.java

```diff
@@ -43,9 +43,16 @@
  * on the data type and partition transform type.
  */
 public class DeltaValueConverter {
+  private static final DateFormat DATE_FORMAT;
```

Comment:
If I remember correctly, SimpleDateFormat is not thread-safe so we can run into issues with a shared instance